src/apps/console/templates/console.html

```diff
@@ -12,159 +12,358 @@
 </pre>
             <br />
             <br />
-            <div class="punch">DataHub is a (GitHub-Like) Data Ecosystem for Individuals, Teams and People.
+            <div class='punch'>DataHub is a (GitHub-Like) Data Ecosystem for Individuals, Teams and People.
             </div>
             <br />
         </div>
     </div>
 </div>
-<script type="text/javascript">
-if (typeof String.prototype.startsWith != 'function') {
-    // see below for better implementation!
-    String.prototype.startsWith = function(str) {
-        return this.indexOf(str) == 0;
-    };
-}
+<script type = 'text/javascript' >
 
-function print_result(res, term) {
-    if (res.num_tuples >= 0) {
-        var col_names = []
-        var separator = []
+var base_url = 'http://' + window.location.host;
+var client_id = '7ByJAnXj2jsMFN1REvvUarQjqXjIAU3nmVB661hR';
+var redirect_uri = 'http://' + window.location.host + '/apps/console/';
+var kCmdList = [
+    'connect &lt;repo-base&gt; \t\t\t  -- connect to a datahub user',
+    'mkrepo &lt;repo-name&gt; \t\t\t   -- create a new repository',
+    'rm &lt;repo-name [-f]&gt; \t\t\t  -- remove a repository',
+    'ls \t\t\t\t\t\t\t   -- list repositories',
+    'ls &lt;repo-name&gt; \t\t\t\t   -- list tables/views in a repository',
+    'desc &lt;repo-name.table-name [-l]&gt;  -- print schema info of a table [with data type]',
```

Comment:
I'd like to see some more API functionality exposed. Could you add listing/describing collaborators?